README.md

```diff
@@ -25,6 +35,28 @@ yarn workspace demo develop
 To browse the demo site, visit [http://localhost:8001](http://localhost:8001) in
 your browser.
 
+## Configuration
+
+You can configure `gatsby-theme-newrelic` for use with any set of docs using the provided configuration options.
+
+## Options
+
+| Option name | Type   | Required | Description                                                                                                                                                                                                                                                                          |
+| ----------- | ------ | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| newrelic    | object | No       | Configurations for `gatsby-plugin-newrelic`. More details for the configuration can be found in [the repo](https://github.com/newrelic/gatsby-plugin-newrelic) for the plugin.                                                                                                       |
+| robot       | object | No       | `robot` options are forwarded to the `gatsby-plugin-robots-txt` plugin. The default value is `policy: [{ userAgent: '*', allow: '/' }],`. More details about robots.txt and configuring the plugin can be [found here](https://www.gatsbyjs.org/packages/gatsby-plugin-robots-txt/). |
+
+## Components
+
+### `Button`
+
+Easily create buttons in different styles.
+
+| Prop    | Type   | Required | Description                                                                                |
+| ------- | ------ | -------- | ------------------------------------------------------------------------------------------ |
+| variant | string | yes      | Configures the variant of the button. Must either be `'plain'`, `'primary'`, or `'normal'` |
```

Comment:
Should we push people to try and use the constants rather than the string values? If so, maybe we should list this as `Button.VARIANT.PLAIN`, `Button.VARIANT.PRIMARY`, and `Button.VARIANT.NORMAL`.

Thoughts?

Reply:
I'm a fan of this pattern.

It usually provides a better error when there's a typo and some editors even provide autocomplete options when writing code that uses the variables. As someone who is typo-prone and lazy, these are both wins for me.

Reply:
Awesome. Let's document them using the constants. Thanks!

Reply:
Okay! I'll make these changes